src/docs/contributing/minimalist-documentation.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Our Approach
+noindex: true
+sidebar_order: 2
+---
+
+Sentry uses a minimalist approach to documentation. Our docs have one goal: help customers gain mastery _quickly_. Minimalism in documentation puts the reader at the center of the content. Our audience is reading the documentation to solve a problem or finalize a setup or to understand. Thus, think **very critically** about the content that is provided, especially on a primary page.
+
+## Use This Test
+
+Ask these questions as you're developing content:
+
+- Is this information critical to helping a developer get up and running? If not, consider moving it to a subpage with a link to it from the primary page (or, as suggested by one of our engineering managers, "cool, I can bookmark this for later").
+
+- Is this obvious from the UI? Our UI is largely self-documenting, so we don't need to explain what's already evident.
+
+## What Goes Where?
+
+Our content has some pretty clear divisions:
+
+- Platform content - instrument and configure your SDK
+- Product content - look at all the data ingested into Sentry
+
+And within those primary categories:
+
+- Reference: What you need to know
+- Action: What you can do (modify, filter, and so forth)
+
+If you're uncertain, ask. The Docs team is happy to discuss.
+
+## Writing for Sentry
+
+Keep these concepts in mind when contributing to Docs:
```

Comment:
```suggestion
Keep these concepts in mind when contributing to docs:
```

As the team name I got the capitalization, but here it is just short for "documentation", right?

Reply:
Yes. Thanks.